src/saltext/vmware/modules/vmc_distributed_firewall_rules.py

```diff
@@ -18,6 +18,17 @@ def __virtual__():
     return __virtualname__
 
 
+def _create_payload_for_distributed_firewall_rule(rule_id, user_input):
+    """
+    This function creates the payload based on the template and user input passed
+    """
+    data = vmc_request.create_payload_for_request(
+        vmc_templates.create_distributed_firewall_rules, user_input
+    )
+    data["id"] = data["display_name"] = rule_id
+    return data
+
+
 def get(
```

Comment:
**question** should this be `list` instead of get?

Reply:
changed to list